components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/hybrid/JdbcHybridRoleManager.java

```diff
@@ -313,6 +320,81 @@ public String[] getUserListOfHybridRole(String roleName) throws UserStoreExcepti
         }
     }
 
+    public String[] getPrimaryRoles() throws UserStoreException {
+        String filter = "%";
```

Comment:
Shall we define this as a constant?

Reply:
Addressed by Commit [909b252](https://github.com/wso2/micro-integrator/commit/909b2525f803b797c9cc9d4a686586259c858013)